source/renderer/app/themes/daedalus/dark-cardano.js

```diff
@@ -785,6 +785,9 @@ export const DARK_CARDANO_THEME_OUTPUT = {
       'rgba(255, 255, 255, 0.5)',
     '--theme-sidebar-category-networkInfo-background-color': '#1fc1c3',
     '--theme-sidebar-category-networkInfo-text-color': '#121326',
+    '--theme-sidebar-sort-button-active-background-color':
+      'rgba(250, 251, 252, 0.1)',
```

Comment:
this color is incorrect (it's light-blue theme color) 
it should be rgba (255, 255, 255, 0.1)